maintainers/maintainer-list.nix

```diff
@@ -1765,6 +1765,11 @@
     github = "Gerschtli";
     name = "Tobias Happ";
   };
+  ggpeti = {
```

Comment:
This should be part of a separate commit. Like `maintainers/maintainer-list: Add ggpeti as maintainer`

Reply:
I wouldn't want to force push to the PR as it broke checks the other day. So I'll have to leave it there, thanks for the tip though.

Reply:
@ggPeti If it breaks checks we'd just have to re-eval them, and breaking them is probably not your fault. We'd prefer if it's separate since the commits become more independent.